verilog/tools/ls/autoexpand.cc

```diff
@@ -89,33 +154,156 @@ class AutoExpander {
   // Token context created from the buffer contents
   const TokenInfo::Context context_;
 
+  // Symbol table wrapper for the language server
+  SymbolTableHandler &symbol_table_handler;
```

Comment:
Since this is something we potentially modify, make it a pointer.  In Verible we like to emphasize if some field points to something that can be modified (pointer) or something only read (const reference).